src/fun/builtins.bend

```diff
@@ -47,6 +47,24 @@ List/filter (List/Cons x xs) pred =
     (List/filter xs pred)
   }
 
+## Recursively sums all elements in a list of natural numbers.
+# If the list is empty, returns 0.
+# List/sum (xs: List(n24)) -> n24
+List/sum (List/Nil) = 0
+List/sum (List/Cons x xs) = (+ x (List/sum xs))
+
+# Generates a list of natural numbers starting from `start` and ending at `end` (inclusive).
+# If the start value is greater than the end value, returns an empty list.
+# List/range (start: n24, end: n24) -> (List n24)
```

Comment:
```suggestion
# Generates a list of unsigned numbers starting from `start` and ending at `end` (inclusive).
# If the start value is greater than the end value, returns an empty list.
# List/range (start: u24, end: u24) -> (List u24)
```
Since you're adding `1`, the input has to be u24